src/logging.h

```diff
@@ -54,6 +54,7 @@ namespace BCLog {
         COINDB      = (1 << 18),
         QT          = (1 << 19),
         LEVELDB     = (1 << 20),
+        DISKFLUSH   = (1 << 21),
```

Comment:
Would prefer adding this to the current `BENCH` category, it includes more timings like this. Adding a new category seems overkill to me.

Reply:
Yeah, I think that makes sense. Fixed.